pkg/cli/zip_per_node.go

```diff
@@ -104,10 +104,10 @@ func (zc *debugZipContext) collectCPUProfiles(
 			var pd profData
 			err := timeutil.RunWithTimeout(ctx, "fetch cpu profile", zc.timeout+zipCtx.cpuProfDuration, func(ctx context.Context) error {
 				resp, err := zc.status.Profile(ctx, &serverpb.ProfileRequest{
-					NodeId:  fmt.Sprintf("%d", nodeID),
-					Type:    serverpb.ProfileRequest_CPU,
-					Seconds: secs,
-					Labels:  true,
+					NodeId:     fmt.Sprintf("%d", nodeID),
+					Type:       serverpb.ProfileRequest_CPU,
+					Seconds:    secs,
+					WithLabels: true,
```

Comment:
for future PRs: please put trivial refactors like this in separate commits.

Reply:
Also, is this a backwards-incompatible change? I believe profiles can be requested from this endpoint via JSON, so this would break. Let's avoid unnecessary renames unless they are confirmed as benign by @cockroachdb/obs-inf-prs.

Reply:
noted, undid this rename